src/next.js

```diff
@@ -0,0 +1 @@
+export * from './components/navigation';
```

Comment:
I tend to avoid `export *` on the basis that it is useful to be able to look at a file and see exactly what it exports. It also makes it easier to see when new things are added/removed from the exported interface when looking through the history.

Reply:
Ah, yes, I totally see that perspective! Happy to expand this, and to remove the somewhat crufty `src/components/navigation/index` module as it doesn't serve a purpose, really. It was hanging around from when I thought I'd be grouping components more "aggressively."

Note I'm proposing still using `export *` for _icons_ because there are so many of them and it's ergonomic to auto-generate a local `index` in the icons directory (see #494)